### DISCUSSION TITLE
Seams appear after importing  an obj file which is exported from blender file

### DISCUSSION THREAD

**Discussion by zaichenhu** [mosra/magnum#586] [Q&A]
Hello, I have got a problem, why some seams appearing after importing an obj file? For example, a monkey exported from blender, it looks like the monkey is jointed at some special surface, are there some flags that I have forgotten to set?

**Comment by mosra** [comment]
I'm not sure what you mean by seams, can you show a screenshot? I assume this is a different problem than the smooth / flat shading we discussed in #585.

---

Btw., I converted this to a discussion as I think discussions are a better place for Q&A than issues. So next time please open a discussion topic instead -- thank you! :) Alternatively, feel free to join the chat at https://gitter.im/mosra/magnum (simply log in with your GitHub account), there it's way less formal and the response time might also be shorter.
